src/IceRpc.Slice.Generators/IceRpc.Slice.Generators.csproj

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+    <IsPackable>false</IsPackable>
+    <developmentDependency>true</developmentDependency>
```

Comment:
Is this correct?
The camel case developmentDependency looks like an attribute name, not a property name.

Reply:
Fixed the case